components/google-cloud/google_cloud_pipeline_components/experimental/evaluation/component.yaml

```diff
@@ -0,0 +1,57 @@
+name: Launch Evaluation Dataflow Flex Template
+description: |
+  Launchs a dataflow job from the evaluation flex template.
+metadata:
+  labels:
+    add-pod-env: 'true'
+inputs:
+  - name: project_id
+    description: 'Required. The ID of the Cloud Platform project that the job belongs to.'
+    type: String
+  - name: location
+    description: 'The regional endpoint to which to direct the request.'
+    default: ''
+    type: String
+  - name: staging_dir
+    description: >-
+      Optional. The GCS directory for keeping staging files.
+      A random subdirectory will be created under the directory to keep job info
+      for resuming the job in case of failure.
+    default: ''
+    type: String
+  - name: wait_interval
+    description: 'Optional wait interval between calls to get job status. Defaults to 30.'
+    default: '30'
+    type: Integer
+  - name: config_uri
+    type: EvaluationConfig
+outputs:
+  - name: output_uri
+    type: Metrics
+  - name: job_id
+    description: 'The id of the created dataflow job.'
+    type: String
+  - name: MLPipeline UI metadata
+    type: UI metadata
+implementation:
+  container:
+    image: gcr.io/ml-pipeline/ml-pipeline-gcp:1.7.0-rc.2
+    args: [
+      --ui_metadata_path, {outputPath: MLPipeline UI metadata},
+      kfp_component.google.dataflow, launch_flex_template,
+      --project_id, {inputValue: project_id},
+      --location, {inputValue: location},
+      --launch_parameters,
+      concat: [
+        '{"container_spec": {"image": "gcr.io/model-evaluation-dev/tfma-cloud-flex:candidate", "sdk_info": {"language": 1}}, "job_name": "xwx-blahblah", "parameters": {"disk_size_gb": "50", "config": "{}", "json_mode": true, "config_gcs_uri": "',
+        {inputUri: config_uri},
+        '", "output_gcs_uri": "',
+        {outputUri: output_uri},
+        '"}}',
+      ],
+      --staging_dir, {inputValue: staging_dir},
+      --wait_interval, {inputValue: wait_interval},
+      --job_id_output_path, {outputPath: job_id},
+    ]
+    env:
+      KFP_POD_NAME: "{{pod.name}}"
```

Comment:
this won't be supported. please drop this.

Reply:
Done.